pkgs/applications/virtualization/qemu/default.nix

```diff
@@ -111,8 +117,14 @@ stdenv.mkDerivation rec {
       sha256 = "sha256-oC+bRjEHixv1QEFO9XAm4HHOwoiT+NkhknKGPydnZ5E=";
       revert = true;
     })
+    # Adds binfmt P support, activated when argv[0] ends with `-binfmt-P`.
+    (fetchpatch {
+      url = "https://salsa.debian.org/qemu-team/qemu/-/raw/master/debian/patches/linux-user-binfmt-P.diff";
```

Comment:
This does not exist anymore, removed in [this commit](https://salsa.debian.org/qemu-team/qemu/-/commit/1f805a1389b80ecda82a07b4ec190a0352fa3eb0).

I don't understand all details here, could this implementation work without the patch as well?

Reply:
[Raw link to the patch](https://salsa.debian.org/qemu-team/qemu/-/raw/7551fa04dd734788faa8d6ee6fec6616dff88331/debian/patches/linux-user-binfmt-P.diff) if we still need it.

Reply:
Yes, it indeed can now and is independent of how the emulator is named. I'll remove this patch.